SDKBuildScripts/xplat_BuildAndTestAndroid.sh

```diff
@@ -33,7 +23,7 @@ BuildAPK() {
     pushd "$AndroidProjectPath"
     ExitIfError
 
-    ./gradlew build
+    ./gradlew assembleDebug
```

Comment:
Should parameterize the gradle task so as to facilitate testing different configurations?

Reply:
Not a bad idea for other purposes, but if we want AppCenter to pass, this is the build parameter we must use.
I think if we set up a script for customer use, then perhaps it should be configurable.
But for now, we're just making the test work, so I'm going to say "won't fix"

Reply:
On this point, I've done some of the intermediate work for this, including, identifying the releaseApkPath, and setting up all the testing to use that path if it exists.
I'm sticking with assembleDebug for now, since it 1. Works, and 2. is faster.